core/src/test/kotlin/com/exactpro/th2/read/db/app/FullReadDbOracleIntegrationTest.kt

```diff
@@ -0,0 +1,308 @@
+                    },
+                    "$PULL_BY_TIMESTAMP_WITH_TIME_ZONE_QUERY_ID": {
+                        "query": "SELECT * FROM $TABLE_NAME WHERE $TIMESTAMP_WITH_TIME_ZONE_COLUMN > ${"$"}{$TIMESTAMP_WITH_TIME_ZONE_COLUMN:timestamp}"
+                    },
+                    "$PULL_BY_TIMESTAMP_WITH_LOCAL_TIME_ZONE_QUERY_ID": {
+                        "query": "SELECT * FROM $TABLE_NAME WHERE $TIMESTAMP_WITH_LOCAL_TIME_ZONE_COLUMN > ${"$"}{$TIMESTAMP_WITH_LOCAL_TIME_ZONE_COLUMN:timestamp}"
+                    }
+                },
+                "publication": {
+                    "maxDelayMillis": 1,
+                    "maxBatchSize": 1
+                },
+                "useTransport": true
+            }
+            """.trimIndent()
+    )
+
+    private fun execute(action: Connection.() -> Unit) {
+        oracle.createConnection("").use { it.action() }
+    }
+
+    private fun Connection.insertData(records: List<Record>) {
+        val prepareStatement = prepareStatement(
+            """
+                INSERT INTO $TABLE_NAME (ID, $DATE_COLUMN, $TIMESTAMP_COLUMN, $TIMESTAMP_WITH_TIME_ZONE_COLUMN, $TIMESTAMP_WITH_LOCAL_TIME_ZONE_COLUMN)
+                VALUES (?, ?, ?, ?, ?)
+            """.trimIndent()
+        )
+        for (record in records) {
+            prepareStatement.setInt(1, record.id)
+            prepareStatement.setTimestamp(2, Timestamp(record.date.toEpochMilli()))
+            prepareStatement.setTimestamp(3, Timestamp(record.timestamp.toEpochMilli()))
+            prepareStatement.setTimestamp(4, Timestamp(record.timestampWithTimeZone.toEpochMilli()))
+            prepareStatement.setTimestamp(5, Timestamp(record.timestampWithLocalTimeZone.toEpochMilli()))
+            prepareStatement.addBatch()
+        }
+        prepareStatement.executeBatch()
+        LOGGER.info { "Inserted $records" }
+    }
+
+    private fun Connection.initTestData() {
+        createStatement()
+            .execute(
+                """
+                    CREATE TABLE $TABLE_NAME   
+                     (
+                      ID INT NOT NULL ,
+                      $DATE_COLUMN DATE NOT NULL ,
+                      $TIMESTAMP_COLUMN TIMESTAMP NOT NULL,
+                      $TIMESTAMP_WITH_TIME_ZONE_COLUMN TIMESTAMP WITH TIME ZONE NOT NULL, 
+                      $TIMESTAMP_WITH_LOCAL_TIME_ZONE_COLUMN TIMESTAMP WITH LOCAL TIME ZONE NOT NULL
+                     )
+                """.trimIndent()
+            )
+        LOGGER.info { "table created" }
+    }
+
+    private fun Connection.dropTable() {
+        createStatement()
+            .execute(
+                """
+                BEGIN
+                   EXECUTE IMMEDIATE 'DROP TABLE $TABLE_NAME';
+                EXCEPTION
+                   WHEN OTHERS THEN
+                      IF SQLCODE != -942 THEN
+                         RAISE;
+                      END IF;
+                END;
+            """.trimIndent()
```

Comment:
note: redundant indentation
```suggestion
                """
                BEGIN
                   EXECUTE IMMEDIATE 'DROP TABLE $TABLE_NAME';
                EXCEPTION
                   WHEN OTHERS THEN
                      IF SQLCODE != -942 THEN
                         RAISE;
                      END IF;
                END;
                """.trimIndent()
```